test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -262,7 +262,9 @@ Describe "New-Item with links fails for non elevated user if developer mode not
         $testlink             = "testlink"
         $FullyQualifiedFile   = Join-Path -Path $TestDrive -ChildPath $testfile
         $TestFilePath         = Join-Path -Path $TestDrive -ChildPath $testlink
-        $developerMode = (Get-ItemProperty HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock -ErrorAction SilentlyContinue).AllowDevelopmentWithoutDevLicense -eq 1
+        $developerModeEnabled = (Get-ItemProperty HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock -ErrorAction SilentlyContinue).AllowDevelopmentWithoutDevLicense -eq 1
+        $minBuildRequired     = ([System.Environment]::OSVersion.Version.Major -eq 10 -and [System.Environment]::OSVersion.Version.Build -ge 14972) -or ([System.Environment]::OSVersion.Version.Major -ge 11)
```

Comment:
couldn't this just be:

```suggestion
        $minBuildRequired     = [System.Environment]::OSVersion.Version -ge "10.0.14972"
```

Reply:
That would probably work better as 10.1.0 wouldn't pass the original check